### ISSUE TITLE
Barcode reader crashes

### ISSUE THREAD

**Issue by powermosfet** [bookwyrm-social/bookwyrm#2592]
**Describe the bug**
When reading a barcode, the code gets recognized properly, but instead of closing the reader and searching for the barcode, a javascript error is printet to the console

**To Reproduce**
1. Go to the front page
2. Click on the barcode button
3. Place a barcode in front of the camera
5. The camera view displays a blue frame around the detected barcode, but nothing happens. The dialog is still open, and the app does not search for the barcode. In the javascript console, there is an error message  "TypeError: statusNode.querySelector(...) is null"

**Expected behavior**
The scanned code should be searched for (and I suppose the barcode dialog should close)

**Screenshots**
![bilde](https://user-images.githubusercontent.com/53771/211766229-8f7c387b-4338-48be-978c-f77e013412d7.png)

Error from javascript console:
```
Uncaught TypeError: statusNode.querySelector(...) is null
    openBarcodeScanner https://bookwyrm-social.sfo3.digitaloceanspaces.com/static/js/bookwyrm.js?v=ad848b97:779
    n https://bookwyrm-social.sfo3.digitaloceanspaces.com/static/js/vendor/quagga.min.js?v=ad848b97:3
    publish https://bookwyrm-social.sfo3.digitaloceanspaces.com/static/js/vendor/quagga.min.js?v=ad848b97:3
    publish https://bookwyrm-social.sfo3.digitaloceanspaces.com/static/js/vendor/quagga.min.js?v=ad848b97:3
    value https://bookwyrm-social.sfo3.digitaloceanspaces.com/static/js/vendor/quagga.min.js?v=ad848b97:3
    value https://bookwyrm-social.sfo3.digitaloceanspaces.com/static/js/vendor/quagga.min.js?v=ad848b97:3
    t https://bookwyrm-social.sfo3.digitaloceanspaces.com/static/js/vendor/quagga.min.js?v=ad848b97:3
    ...
    t https://bookwyrm-social.sfo3.digitaloceanspaces.com/static/js/vendor/quagga.min.js?v=ad848b97:3
    (Async: FrameRequestCallback)
    t https://bookwyrm-social.sfo3.digitaloceanspaces.com/static/js/vendor/quagga.min.js?v=ad848b97:3
```

**Instance**
bookwyrm.social

**Additional context**
It happens consistently across different devices (phone/desktop) and browsers

---

**Desktop (please complete the following information):**
    - OS: Mac OS Ventura 13
    - Browser: Firefox
    - Version: 108.0.2

**Smartphone (please complete the following information):**
    - Device: Huawei Mate 10 Pro
    - OS: Android EMUI 12.0.0
    - Browser: Chrome
    - Version: 108.0.5259.128

**Comment by hughrun** [comment]
I think we have a fix for this at #2605 - I'll test it later today.

**Comment by hughrun** [comment]
@mouse-reeve this can be closed now as #2605 has been pulled (but didn't include a reference to this issue)
